frontend/src/app/shared/components/work-package-graphs/embedded/wp-embedded-graph.component.ts

```diff
@@ -98,10 +98,32 @@ export class WorkPackageEmbeddedGraphComponent {
   }
 
   protected setChartOptions() {
+    const bodyFontColor= getComputedStyle(document.body).getPropertyValue('--body-font-color');
+    const gridLineColor= getComputedStyle(document.body).getPropertyValue('--borderColor-default');
```

Comment:
I know that I brought this example of th doughnut chart up, but I just checked on edge and there, the grid and the scales are not visible at all for these kind of diagrams. There are only shown for some (like the bar chart)

<img width="671" alt="Bildschirmfoto 2024-07-04 um 08 44 29" src="https://github.com/opf/openproject/assets/7457313/121da184-6140-42a7-87d6-c304cd268e2e">